src/test/java/lesson3/task1/SeatTest.java

```diff
@@ -0,0 +1,39 @@
+package lesson3.task1;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SeatTest {
+
+    @Test
+    void shouldReserveSeat() {
+        //given
+        Seat reservedSeat = new Seat(1, 2, false);
+        Seat availableSeat = new Seat(2, 3, true);
+        //when
+        String isReserved = Seat.reserveSeat(reservedSeat);
+        String isAvailable = Seat.reserveSeat(availableSeat);
+        //then
+        Assertions.assertSame(isReserved, "Sorry, this seat is not available, try another one please.");
```

Comment:
as I mentioned, to check weather change was succesfull or not i have to check if the result is equal to sam loooong message. So much easier would be to check if(isReserved == true) or if(isReserved) those two examples will do the same when isReserved method will be of boolean type